httpserver/service_node.cpp

```diff
@@ -292,6 +293,14 @@ parse_swarm_update(const std::shared_ptr<std::string>& response_body) {
     return bu;
 }
 
+int ServiceNode::valid_pubkey_length() const {
```

Comment:
I think returning `size_t` is slightly more appropriate for a length.
Especially if you already use it in the `deserialize_messages` signature.

Reply:
I think we could have a `const size_t pubkey_length` member that is initialised in the constructor.
Then `valid_pubkey_length()` would just return the value of that member.
This would better transpire that the value doesn't change within the execution lifetime, and also prevent reading the value of `testnet_` every time.